Update index.md (#3395)
MicrosoftDocs/windows-uwp: hub/apps/winui/index.md

| **[WinUI 3](#windows-ui-3-library)** ships with the [Windows App SDK](../windows-app-sdk/index.md). | **[WinUI 2](#windows-ui-2-library)**, the 2nd generation of WinUI, ships as a standalone [NuGet package](https://www.nuget.org/packages/Microsoft.UI.Xaml/), and is integrated with [Windows 10 and later SDKs](https://developer.microsoft.com/windows/downloads/windows-10-sdk/). |
| UX stack and control library completely decoupled from the OS and [Windows 10 and later SDKs](https://developer.microsoft.com/windows/downloads/windows-10-sdk/), including the core framework, composition, and input layers of the UX stack.                                                                        | UX stack and control library tightly coupled to the OS and [Windows 10 and later SDKs](https://developer.microsoft.com/windows/downloads/windows-10-sdk/).                                                                                                                                                                                                                                                                                                                                                          |

> | WinUI 3 can be used to build production-ready **desktop/Win32** Windows apps. | WinUI 2 can be used in UWP applications and incorporated into new or existing desktop applications using [XAML Islands](../desktop/modernize/xaml-islands.md) (for installation instructions, see [Getting started with the WinUI 2 Library](winui2/getting-started.md)). |

Reviewer: Nit: Drop "new and existing" as you're just repeating all possibilities.

```suggestion
| WinUI 3 can be used to build production-ready **desktop/Win32** Windows apps. | WinUI 2 can be used in UWP applications and incorporated into desktop applications using [XAML Islands](../desktop/modernize/xaml-islands.md) (for installation instructions, see [Getting started with the WinUI 2 Library](winui2/getting-started.md)). |
```

| WinUI 3 ships as a component of [the Windows App SDK](../windows-app-sdk/index.md) framework package, with Visual Studio project templates in the Windows App SDK Visual Studio Extension (VSIX). | Part of WinUI 2 ships within the operating system itself (the Windows.UI.* family of UWP WinRT APIs) and part of it ships as a library (“Windows UI Library 2”) with additional controls, elements and the latest styles on top of what’s already included in the operating system itself. With WinUI 2, these features ship in a downloadable NuGet package. However, other significant parts of the UI stack are still built-in to the OS, such as the core XAML framework, input, and composition layers. |

> | WinUI 3 supports C# .NET 5 and C++ for desktop apps. | WinUI 2 supports C# .NET Native and C++ apps. |

Author: ```suggestion
| WinUI 3 supports C# (.NET 5) and C++ for desktop apps. | WinUI 2 supports C# and Visual Basic (.NET Native), and C++ apps. |
```

| WinUI 3 support for production-ready UWP apps is currently in preview, see [WinUI 3 - Project Reunion 0.5 Preview](winui3/release-notes/winui3-project-reunion-0.5-preview.md).                                                                                                                                | WinUI 2 can be incorporated into production UWP apps by installing a NuGet package into a new or existing UWP project. WinUI controls and styles can then be referenced directly in new apps, or by updating "Windows.UI." namespace references to "Microsoft.UI." in existing apps.                                                                                                                                                                                    |

> | WinUI 3 supports the Chromium-based [WebView2](/microsoft-edge/webview2/) control |  WinUI 2 supports the  [WebView](/windows/uwp/design/controls-and-patterns/web-view) control on all devices, and the [WebView2](/microsoft-edge/webview2/) control for Desktop only. |

Reviewer: Nit: Grammar tweak

```suggestion
| WinUI 3 supports the Chromium-based [WebView2](/microsoft-edge/webview2/) control |  WinUI 2 supports the  [WebView](/windows/uwp/design/controls-and-patterns/web-view) control on all devices, and the [WebView2](/microsoft-edge/webview2/) control on desktop. |
```

Author: ```suggestion
| WinUI 3 supports the Chromium-based [WebView2](/microsoft-edge/webview2/) control |  WinUI 2 supports the  [WebView](/windows/uwp/design/controls-and-patterns/web-view) control on all devices, and starting with WinUI 2.8 prerelease, the [WebView2](/microsoft-edge/webview2/) control on Desktop. |
```
